examples/inference/colossal_llama2_test.py

```diff
@@ -0,0 +1,84 @@
+import os
```

Comment:
may need move this to 'test' folder and change a name

Reply:
OK!

Reply:
fixed it.